tests/test_odin_sequencer.py

```diff
@@ -516,10 +530,7 @@ def test_execute_when_module_is_modified_while_auto_reload_enabled(shared_datadi
     _await_queue_size(manager, 1)
 
     message = manager.execute('generate_message')
-    basic_seq_value = manager.basic_sequence(0)
-
     assert message == 'Message: Hello World'
-    assert basic_seq_value == 0
```

Comment:
The assertion here that was introduced in commit [4067225](https://github.com/stfc-aeg/odin-sequencer/pull/12/commits/4067225bf2b60a4a4ee867bd5a81c5be5ee70c8e) in PR #12 , was moved to a new test method `test_attribute_func_when_module_sequence_is_added_auto_reload_enabled` because the test here tests the `execute` function when a module is modified while auto-reload is enabled.